src/services/JobService.js

```diff
@@ -81,13 +81,29 @@ async function _validateSkills (skills) {
   }
 }
 
+/**
+ * Validate by exact match on name field if all skills exist.
+ *
+ * @param {Array} skills the list of skill names to validate
```

Comment:
Update param docs using the JSDoc format.

Reply:
Fixed